keras-dreambooth-sprint/README.md

```diff
@@ -62,13 +64,13 @@ prompt = f"A photo of {unique_id} {class_label} in a bucket"
 image = pipeline(prompt, num_inference_steps=50).images[0]
 ```
 
-### **Model Hosting**
+### Model Hosting
 
-At the end of this notebook you will see a section dedicated for hosting, and a separate one for inference. We will be using huggingface_hub library’s Keras-specific model pushing and loading functions, `push_to_hub_keras` and `from_pretrained_keras` . We will first push the model using `push_to_hub_keras`. After model is pushed, you will see the model is hosted with a model card like below 👇 
+At the end of this notebook you will see a section dedicated for hosting, and a separate one for inference. We will be using the `huggingface_hub` library’s Keras-specific model pushing and loading functions: `push_to_hub_keras` and `from_pretrained_keras` . We will first push the model using `push_to_hub_keras`. After model is pushed, you will see the model is hosted with a model card like below 👇 
```

Comment:
Which notebook are we referring to here? I am mentioning since this is being pointed to from a guide and not a notebook IIUC.

Reply:
Addressed it